Application/Services/AuthService.cs

```diff
@@ -53,18 +54,18 @@ public async Task<AuthResponseDto> RegisterAsync(RegisterRequestDto request)
         var existingEmail = await userRepository.GetByEmailAsync(request.Email);
         if (existingEmail is not null)
         {
-            throw new EmailAlreadyUsedException();
+            throw new AppException(400, RegisterErrorCode.EMAIL_ALREADY_USED);
```

Comment:
Wcześniej `EmailAlreadyUsedException` rzucał `409`

```suggestion
            throw new AppException(409, RegisterErrorCode.EMAIL_ALREADY_USED);
```